client/src/globalState.tsx

```diff
@@ -328,7 +328,7 @@ export const GlobalStateProvider: React.FC<GlobalStateProviderProps> = ({
     userEmail,
     userLookup,
 
-    activeTab,
+    activeTab: payloadToBeSent ? ChatTab : activeTab,
```

Comment:
I don't like this, this stops you switching tabs entirely when you have a payload ready to send without any ui feedback. I think it'd be better to set the chat as the active tab when a payload is added but after that allow the user to switch back and forth as they want.

Reply:
yeah good point, I've replaced with a `useEffect` which is dependent on `payloadToBeSent` which should have the behaviour you suggested

Reply:
deployed to CODE and works as you described. It does make me wonder though if activeTab should be global state or per pinboard (since its confusing if you navigate up from a pinboard which is on asset tab then open a new pinboard it opens in asset view, which could be confusing as they look similar - one might not think there are any messages)

Reply:
agreed, it should definitely at least appear to be a per-pinboard state (iirc it's a little tricky to actually do that as the navigation is sibling to the pinboards, so the state needs to live in panel which is effectively global. we should be able to get the same effect with a well-placed useEffect though)

Reply:
> agreed, it should definitely at least appear to be a per-pinboard state (iirc it's a little tricky to actually do that as the navigation is sibling to the pinboards, so the state needs to live in panel which is effectively global. we should be able to get the same effect with a well-placed useEffect though)

done that too now @andrew-nowak